src/features/Tiles/BigMovieTile/index.js

```diff
@@ -14,12 +16,12 @@ import {
     Description,
 } from "./styled";
 
+export const BigMovieTile = () => {
+    const movieData = useSelector(selectMovieData);
 
-export const BigMovieTile = ({ movieData }) => {
-    const image = movieData.poster_path;
```

Comment:
i find this change negative - if We`d like to change api source, we need to check all component to find where it s used. Maybe its better to declare image and than use it where needed?

Reply:
Yes, you are right. I don't know why it wasn't working yesterday 🤣 . "const image" is back now. 
BUT I've discovered another problem. Check out how the poster act on different screens. Sometimes you can't see the whole picture. Do you think it should be this way?

Reply:
Ive noticed it. but i think its just a new problem to resolve, so I add it to trello.